core/runtime/src/impl/EggObjectUtil.ts

```
@@ -26,4 +26,146 @@
+        Object.setPrototypeOf(obj, v);
+        return true;
+      },
+    });
+    return proxy;
+  }
+
+  static contextEggObjectProxy(proto: EggPrototype, objName: PropertyKey): PropertyDescriptor {
+    const PROTO_OBJ_PROXY = Symbol(`EggPrototype#objProxy#${String(objName)}`);
+    if (!proto[PROTO_OBJ_PROXY]) {
+      proto[PROTO_OBJ_PROXY] = new Proxy({}, {
+        defineProperty(_target: {}, property: string | symbol, attributes: PropertyDescriptor): boolean {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)